arviz/tests/test_plots.py

```diff
@@ -232,16 +232,20 @@ def test_plot_parallel_raises_valueerror(df_trace):  # pylint: disable=invalid-n
         plot_parallel(df_trace)
 
 
-@pytest.mark.parametrize("model_fit", ["pymc3_fit", "stan_fit"])
-def test_plot_parallel(models, model_fit):
+@pytest.mark.parametrize(
+    "model_fit", ["pymc3_fit", "stan_fit"],["normal", "minmax", "rank"]

```

Comment:
The error is coming from here. Because `norm_method` is an argument the tester is looking for it in either a fixture or in parameterization.  

To debug faster try running the tests locally to see what fails. I would look at the plotting tests to see a pattern where we pass multiple kwargs to one function

Reply:
I wrote the initial test following the pattern of `test_plot_joint` which has a similar situation with their parameter `kind`.